example/programs/topic-subscriber/src/main.rs

```diff
@@ -0,0 +1,57 @@
+mod models;
+
+use futures::StreamExt;
+use momento::{topics::Subscription, CredentialProvider, MomentoError, TopicClient};
+use std::{env, error::Error};
+use tracing::{error, info};
+
+use crate::models::MomentoModel;
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn Error>> {
+    tracing_subscriber::fmt()
+        .with_max_level(tracing::Level::INFO)
+        .with_target(false)
+        .json()
+        .init();
+
+    let momento_key = env::var("MOMENTO_API_KEY").expect("MOMENTO_API_KEY is required");
+    // create a new Momento client
+    let topic_client = match TopicClient::builder()
+        .configuration(momento::topics::configurations::Laptop::latest())
+        .credential_provider(CredentialProvider::from_string(momento_key).unwrap())
+        .build()
+    {
+        Ok(c) => c,
+        Err(_) => panic!("error with momento client"),
```

Comment:
Swallowing an error which may tell you what is wrong is not a desirable pattern to model. You should replace this whole match expression with a `?` and let the error propagate naturally.

Reply:
Got it.  Making the change.  Good catch